### PR TITLE
[16.0][ADD] pos_cash_control_override: New module

### PR DESCRIPTION
**Pull request by chienandalu**
From [this change](https://github.com/odoo/odoo/pull/130275) a Point of Sale user who needs to perform cash control operations *Accounting/Billing* permission is required. That's a huge permission for the kind of user that only does cashing machine operations.

We're recovering the former behavior present in previous versions.

This module will be unnecessary if https://github.com/odoo/odoo/pull/188798 gets  merged or the former permissions paradigm is restored in any other way.

cc @Tecnativa TT51987

please review @pedrobaeza @pilarvargas-tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Code review

Is it possible to create some tests to avoid problems in the future?

**Comment by chienandalu** [comment]
I'm going to check if I can import a tour from the core module

**Review by pedrobaeza — APPROVED** [review]
Great!

Code review

File: pos_cash_control_override/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+After this module is installed:
+
+1. Choose a user with all the permissions execpt *Point of Sale/User*.
```
Comment (pilarvargas-tecnativa) [inline pos_cash_control_override/readme/USAGE.md]:
```suggestion
1. Choose a user with all the permissions except *Point of Sale/User*.
```

Comment (chienandalu) [reply pos_cash_control_override/readme/USAGE.md]: fixed :+1:

**Comment by pilarvargas-tecnativa** [comment]
Testing in runboat, if I access with the Demo user without billing permissions but with the pos user permission, when accessing pos, it does not load anything. I don't know if it's a runboat issue.

image
image

**Comment by chienandalu** [comment]
> I don't know if it's a runboat issue.

Did you check with just this module loaded?

**Comment by pilarvargas-tecnativa** [comment]
> Did you check with just this module loaded?

No, do not uninstall anything in runboat. All modules were installed. If it is necessary to test it alone then I will do the local tests.

**Comment by chienandalu** [comment]
As for othe modules in this repo I can't be my life on them behaving friendly with this feature :sweat_smile: but it stands for itself

**Comment by pedrobaeza** [comment]
You can try in runboat in the base database and installing it alone.

**Review by pilarvargas-tecnativa — APPROVED** [review]
:+1:

**Pull request merged**